scrapper/src/profile/posts.js

```diff
@@ -53,6 +54,8 @@ export async function getPosts(userHandle, authorProfileId, start = 0, count = 2
       }
     }
 
+    const comments = await getComments(userHandle,socialDetail[6],socialDetail[5],numComments,1);
```

Comment:
Instead of using a fixed value for "numReplies," check if there is a property or variable related to this number. If such a property doesn't exist, choose a number that won't lead to inaccuracies.